tg/models.py

```diff
@@ -217,6 +226,114 @@ def copy_msgs_text(self) -> bool:
         copy_to_clipboard("\n".join(buffer))
         return True
 
+    def get_chat_info(self, chat: Dict[str, Any]) -> Dict[str, Any]:
```

Comment:
this function looks  too long, i suggest split it.

Reply:
Agree